app/src/main/java/io/neurolab/program_modes/VJ.java

```diff
@@ -0,0 +1,64 @@
+package io.neurolab.program_modes;
+
```

Comment:
@jddeep you can remove blank spaces from here.

Reply:
> @jddeep Good work. Rest everything is fine and good to go. Are you going to update the PR after 208, 209 and 210 get merged (as they have cross dependencies upon each other)? Also, use the reformat and rearrange code options in Android Studio as they would save you from a ton of headache of doing it manually and precisely at every line.

Yes @suresh-gandhi I need to work on #208 #209 @210 first before getting this merged.